.github/workflows/install_test.yaml

```diff
@@ -17,19 +17,33 @@ jobs:
         python_version: ["3.8", "3.9", "3.10"]
     runs-on: ${{ matrix.os }}
     steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.pull_request.head.ref }}
+          repository: ${{ github.event.pull_request.head.repo.full_name }}
       - name: Set up python ${{ matrix.python_version }}
-        uses: actions/setup-python@v2
+        uses: actions/setup-python@v4
         with:
           python-version: ${{ matrix.python_version }}
-      - name: Checkout repository
-        uses: actions/checkout@v2
+          cache: 'pip' 
+          cache-dependency-path: 'pyproject.toml'
+      - uses: actions/cache@v3
+        id: cache
+        with:
+          path: ${{ env.pythonLocation }} 
+          key: ${{ matrix.os- }}-${{ matrix.python_version }}-install-${{ env.pythonLocation }}-${{ hashFiles('**/pyproject.toml') }}-v01
       - name: Build woodwork package
-        run: make package
-      - name: Install complete version of woodwork
         run: |
-          python -m pip install --upgrade pip
-          pip config --site set global.progress_bar off
-          python -m pip install "unpacked_sdist/[complete]"
+          make package
+      - name: Install woodwork with no dependencies
```

Comment:
Just curious, should the name include whether it was retrieved from a cache or not? Would it be useful for potential debugging?

Reply:
Fixed:
https://github.com/alteryx/woodwork/pull/1631/commits/c8b01661ec5415a36cb41b2dc53d0c0575364e2c